cmd/git-xargs_test.go

```diff
@@ -22,6 +24,8 @@ func TestHandleRepoProcessing(t *testing.T) {
 	testConfig.CommitMessage = "test-commit-name"
 	testConfig.Args = []string{"touch", "test.txt"}
 	testConfig.GithubClient = mocks.ConfigureMockGithubClient()
+	testConfig.PullRequestRetries = 0
```

Comment:
Did you mean to not retry during the test?

Reply:
I was initially suspecting something about the new retry logic was borking our tests in CI, so I tried specifying that pull requests that fail due to rate limits should not be retried. But no joy.